templates/takeovers/_improving_productivity_kubernetes.html

```diff
@@ -0,0 +1,35 @@
+<section class="p-strip--image is-dark is-deep u-image-position js-takeover p-improving-prod-takeover">
+      <div class="row u-equal-height">
+        <div class="col-6" >
+          <h1 style="font-weight: 100;" >Improving productivity with Ubuntu Kubernetes</h1>
+          <p class="p-heading--four">Canonical helped investment company, Allan Gray, deploy its Ubuntu Kubernetes cluster in under 2 weeks.</p>
+          <p class="u-hide--small"><a href="/engage/kubernetes-case-study?utm_source=takeover&utm_medium=takeover&utm_campaign=2)FY19_Cloud_K8_CStudy_AllanGray" class="p-button--neutral u-no-margin--bottom" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Improving Productivity - Kubernetes takeover', 'eventLabel' : 'Download the case study', 'eventValue' : undefined });">Download the case study</a></p>
+        </div>
+        <div class="col-6 u-vertically-center u-align--center">
+          <div class="u-sv3">
+            <img src="{{ ASSET_SERVER_URL }}1db850da-K8s+logo.svg" alt="Kubernetes logo" class="p-improving-prod-takeover__image"/>
+          </div>
+          <p class="u-hide--large u-hide--medium"><a href="/engage/kubernetes-case-study?utm_source=takeover&utm_medium=takeover&utm_campaign=2)FY19_Cloud_K8_CStudy_AllanGray" class="p-button--neutral" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Homepage Link', 'eventAction' : 'Improving Productivity - Kubernetes takeover', 'eventLabel' : 'Download the case study', 'eventValue' : undefined });">Download the case study</a></p>
+        </div>
+      </div>
+  <style>
+    .p-improving-prod-takeover__image {
+      width: 240px;
+      height: 240px;
+    }
+    
+    .p-improving-prod-takeover {
+      background: #173D8B;
+      background-image: linear-gradient(-74deg, #173D8B 8%, #326DE6 92%);
+    }
+
+    @media screen and (min-width: 874px) {
+      .p-improving-prod-takeover{ 
+        background-image: url("{{ ASSET_SERVER_URL }}0bfde879-suru-left-side.png"), url("{{ ASSET_SERVER_URL }}4c072e93-suru-right-side.png");
```

Comment:
The linear gradient is getting overwritten here. Add it back to this background-image like so:
`url("{{ ASSET_SERVER_URL }}0bfde879-suru-left-side.png"), url("{{ ASSET_SERVER_URL }}4c072e93-suru-right-side.png"), linear-gradient(-74deg, #173D8B 8%, #326DE6 92%)`

Reply:
@Caleb-Ellis Thanks. Also we had to add to a background-blend-mode. Cheers.